wpilib/tests/test_watchdog.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from unittest.mock import MagicMock
+
+
+def test_init1(wpilib, sim_hooks, MockNotifier):
+    sim_hooks.time = 4.5
+    doggo = wpilib.Watchdog(2)
```

Comment:
I like it, but not sure how @virtuald feels about the 'informal' nature of this wording

Reply:
It's in the tests, who cares :stuck_out_tongue: